test/unit/lib/email-service-spec.js

```diff
@@ -66,6 +66,22 @@ describe('Email Service', () => {
       EmailService.prototype._includeDate.should.not.have.been.called;
     });
 
+    it('sets subject to email-service instance if passed object', () => {
+      const subject = {
+        customer: '',
+        caseworker: ''
+      };
+      emailService = new EmailService({data, subject});
+      emailService.subject.should.be.equal(subject);
```

Comment:
hmm, I s'pose you're right. It tests that the object passed to the class is the object set on the class with the same name.